test/common/network/udp_listener_impl_test.cc

```diff
@@ -152,23 +93,19 @@ TEST_P(UdpListenerImplTest, UseActualDstUdp) {
                 nullptr, false);
 
   const int client_sockfd = client_socket->ioHandle().fd();
-  sockaddr_storage server_addr;
-  socklen_t addr_len;
-
-  getSocketAddressInfo(*client_socket.get(), server_ip->port(), server_addr, addr_len);
+  const sockaddr* server_addr = server_socket->localAddress()->sockAddr();
+  const socklen_t addr_len = server_socket->localAddress()->sockAddrLen();
   ASSERT_GT(addr_len, 0);
 
   // We send 2 packets
   const std::string first("first");
   const std::string second("second");
 
-  auto send_rc = ::sendto(client_sockfd, first.c_str(), first.length(), 0,
-                          reinterpret_cast<const struct sockaddr*>(&server_addr), addr_len);
+  auto send_rc = ::sendto(client_sockfd, first.c_str(), first.length(), 0, server_addr, addr_len);
```

Comment:
Same request to move sendto() to IoHandle

Reply:
ack